main.go

```diff
@@ -335,3 +325,34 @@ func main() {
 		log.Fatal(err)
 	}
 }
+
+// If we are running in WASM/JS, then the browser does all relevant tile caching for us.
+// If running in desktop app mode, we need to cache the tiles ourselves
+func tileProviderForOS() (slippymap.TileProvider, error) {
+	if runtime.GOOS == "js" || false {
```

Comment:
May I ask what the `|| false` is for?

Reply:
it is definitely not meant to be there and definitely does nothing 🙃 I think I was testing the build behaviour of `wasmserve` and just needed to force a rebuild by making a no-op change to the code.

Reply:
Haha no worries. Will you update the branch in the other PR to remove it?